### ISSUE TITLE
Instalação de Componentes Visuais

### ISSUE THREAD

**Issue by nyeeldzn** [HashLoad/boss#155]
Tenho utilizando o boss para controle de versionamente de bibliotecas no meu projeto, estou achando sensacional, porém estou tendo problemas com uma biblioteca pessoal... eu faço o install dela através do link do bitbucket, ele baixa, configura os caminhos de busca, porém aparentemente os componentes visuais não estão sendo compilados/instalados. 
O que pode ser?

**Comment by viniciussanchez** [comment]
Componente visual precisa ter o boss-ide instalado

**Comment by nyeeldzn** [comment]
Obrigado por responder tão rápido!
É necessário apenas instalar ou preciso fazer alguma configuração/procedimento extra?

**Comment by viniciussanchez** [comment]
Sim, você teria que instalar o boss-ide para conseguir fazer a instalação de componentes visuais usando o boss
